VictoriaBankFinalBeta.rb

```diff
@@ -43,35 +44,35 @@
 
 # Go to transaction page
 browser.link(href: '#menu/MAIN_215.CP_HISTORY').click
-sleep(2)
+sleep 2
 browser.input(name: 'from').click
 # Set period from two month early
 (1..2).each { |i| browser.a(title: '< Prev').click }
 # Get current date
-puts current_date = Time.now.strftime('%-d')
+current_date = Time.now.strftime('%-d')
 browser.a(text: current_date).click
 full_account_array = []
 # Search transaction for each account
 (0...accounts_arr.size).each do |i|
-  iban = accounts_arr[i][0]
+  iban = accounts_arr[i].name
   transaction_arr = []
   browser.div(class: 'chosen-container chosen-container-single contract-select chosen-container-single-nosearch').click
   browser.span(text: iban).click
 
   # Wait for page charging.
-  sleep(10)
+  sleep 10
   # Initializing variables.
   day_, date, description, amount, currency = ''
   # Fetch and parse HTML document
   doc = Nokogiri::HTML(browser.html)
-  doc.css('//div[@class="operations"]').each do |div|
-    div.search('div[@class="month-delimiter"]').each do |month_div|
+  doc.css('div.operations').each do |div|
+    div.search('div.month-delimiter').each do |month_div|
       month_year = month_div.text.strip
-      div.search('div[@class="day-operations"]').each do |span|
-        span.search('div[@class="day-header"]').each do |day|
+      div.search('div.day-operations').each do |span|
+        span.search('div.day-header').each do |day|
           day_ = day.text.strip.gsub(/[^\d]/, '')
         end
-        span.search('ul[@class="operations-list"]/li').each do |list|
+        span.search('ul.operations-list/li').each do |list|
           list.search('span').each do |cell|
             time = cell.text.strip if cell.attr('class') == 'history-item-time'
             date = "#{day_} " + "#{month_year}" + "#{time}"
```

Comment:
Need to convert `date` value to `json` format and without time.
`2019-10-02`

Reply:
I changed it. Something else?

Reply:
Write tests for your code. 
I opened an issue.
It is the last task. 
Good luck.
https://github.com/butnaru1/TestTask/issues/6